UltraStar Play/Assets/Scenes/SongSelect/SongRoulette/SongRouletteControl.cs

```diff
@@ -93,6 +94,19 @@ private SongMeta SelectedSongMeta
 
     private void Start()
     {
+        Selection.Subscribe(selection =>
```

Comment:
- What are you trying to achieve? I think the coroutine is not needed
    - For example, UniRx has methods to do stuff when the value has been stable some time ( search `.Throttle(` )